doc/howto/binder-badge-permissions.yaml

```
@@ -0,0 +1,28 @@
+#./github/workflows/binder-badge-permissions.yaml
+name: Binder Badge
+on: [pull_request]
+
+jobs:
+  binder:
+    if: |         
+        ((github.event.issue.author_association != 'OWNER') ||
+        (github.event.issue.author_association != 'COLLABORATOR') ||
+        (github.event.issue.author_association != 'CONTRIBUTOR') ||
+        (github.event.issue.author_association != 'MEMBER'))
```

This logic seemed a bit off, what about the following? I also fixed the indentation to follow two spaces systematically.

```suggestion
      (
        (github.event.issue.author_association == 'OWNER') ||
        (github.event.issue.author_association == 'COLLABORATOR') ||
        (github.event.issue.author_association == 'CONTRIBUTOR') ||
        (github.event.issue.author_association == 'MEMBER')
      )
```

---

I fixed this with a recent commit

---

Note that I switched `!=` to `==`, it's currently still `!=` rather than `==` used.

The logic should be: `if A, B, C, or D`, but currently it is: `if (not A), (not B), (not C), or (not D)`, and the latter will always be true.

---

@hamelsmu final fix here!

---

OH wow, thanks for catching that, yes you are correct.

---

Fixed it